examples/http-server/main_test.go

```diff
@@ -20,9 +21,12 @@ import (
 	"gofr.dev/pkg/gofr/testutil"
 )
 
-const host = "http://localhost:9000"
-
 func TestIntegration_SimpleAPIServer(t *testing.T) {
+	host := "http://localhost:9000"
```

Comment:
Thanks for removing the package level const.

It's way clearer now.

One question, do we really need to have localhost:9000 in the 3 integration tests?

Can't it be moved to a random ports like the other tests?

Reply:
In the` main.go` file, the same server is added as an external `httpService` for demonstration purposes in one of the examples. To keep the setup straightforward, the port has been hardcoded since dynamically altering the `httpService` URL during tests is not feasible in this case.